wiki/Ranking_Criteria/Song_Content_Rules/en.md

```diff
@@ -12,6 +12,52 @@ Tracks with explicit lyrics can still be used so long as they do not overtly vio
+* Explicit or highly suggestive sexual posturing in any capacity
+  - Example: a character in a state of near or partial undress lying atop a bed or another surface with obvious sexual intent/anticipation
+* Depictions of any sexual content involving or targeted at minors, or fantasy characters of uncertain/dubious age
+  - Example: Any form of overt sexual posturing, innuendo or portrayal from an anime character in the 'loli' style, regardless of their established narrative age
+* Erotic nudity or graphic sexual depiction
+  - Example: Active sexual intercourse, foreplay, full frontal nudity with genitals clearly visible
+* Depictions of drug paraphernalia or consumption of any common contraband substance (cocaine, amphetamines, etc), regardless of legality in your host country
+  - Example: bongs, coke spoons, used needles, obvious 'pockmark' bruises suggestive of heavy drug use
+* Depictions of widely abhorrent cultural/religious icons intended to deliberately inflame, such as offensive use of Nazi/fascist aligned swastikas, anti-Semetic terminology and symbols, or any other icon or motif that is an active signet or marker of discrimination against a social, racial or political group
+  
+Visual content **may contain some** of the following aspects in limited quantities, subject to common sense and good faith:
+
+* Depictions of swimsuit or 'scant' dress/undress so long as such imagery is not highly excessive or in violation of the posturing & innuendo rules listed above
+  - Good example: A particularly busty anime character enjoying some time on a dock with her seagull companion
+  - Unacceptable example: Three very busty anime characters wearing similar swimsuits with their breasts poised in obvious presentation, with said imagery occupying more than a third of the image's total scene
+* Imagery of mild to moderate violence without excessive gore or bloodshed
+  - Good example: Two characters from a boxing anime actively fighting one another with bruises, swelling, and some blood present
+  - Unacceptable example: A sword fighter removing a limb from an opponent's body with much blood present
+* Tasteful depiction of sexual "assets" not done to the extremes
+  - Good example: Motion-oriented clothing movement that exposes the front or back of the upper thighs, clothing that exposes a reasonable amount of cleavage, shirtless male characters (with or without nipples)
+  - Unacceptable example: Both genders attired in microkini swimsuits, pasties or other stickers, bandages or adhesive material covering genitals
+* Depiction of obviously artistic nudity within the context of a given work or domain
+  - Good example: Pairing classic Salvador Dali art containing mild nudity to a work cited to be inspired by said piece, or that otherwise houses an obvious, discernible reference to it
+  - Bad example: Using a nude model or figure with a work that has no obvious link to the piece, such as on an unrelated anime opening or ending theme
```

Comment:
"Unacceptable" was used instead of "Bad" above this, should probably have these consistent since they're used to convey the same things (whether that be changing these or the others).